react/nextJsQuestions.md

```diff
@@ -0,0 +1,37 @@
+# Next.js Interview Questions
+
+## General Questions:
+
+[What is Next.js, and how does it differ from traditional React applications?](https://nextjs.org/docs/getting-started/introduction)
+
+[Explain the concept of server-side rendering (SSR) in Next.js](https://nextjs.org/docs/basic-features/data-fetching)
+
+[How does Next.js handle client-side routing?](https://nextjs.org/docs/navigation/introduction)
+
+[What are the benefits of using Next.js for building React applications?](https://nextjs.org/learn/basics/why-use-nextjs)
```

Comment:
For now learning process is structured in other way because of app router. We can refer to https://nextjs.org/docs page or find custom topic on medium.com/anysite.com. Something like this https://ninetailed.io/blog/next-js-vs-react/